src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -16,46 +16,6 @@
   private MobileAdsDataProvider() {
   }
 
-  @DataProvider
-  public static Object[][] leaderboardAndPrefooterSlots() {
```

Comment:
The reasoning behind removing the test cases here has been discussed on slack: https://wikia.slack.com/archives/G08GM4N05/p1519997694000386